src/main/java/seedu/address/logic/commands/DownloadSelectNotesCommand.java

```diff
@@ -64,12 +67,6 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
         }
 
         WebDriver driver = initializeWebDriver();
-        Path notesFolder = Paths.get("notes");
-        try {
-            createDirectoryIfMissing(notesFolder);
```

Comment:
Good that you removed this part from this class, which is in the `Logic` component